src/test/java/org/jabref/logic/integrity/FileCheckerTest.java

```diff
@@ -0,0 +1,58 @@
+package org.jabref.logic.integrity;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.metadata.FilePreferences;
+import org.jabref.model.metadata.MetaData;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+
+public class FileCheckerTest {
+
+    private FileChecker checker;
+    private BibEntry entry;
+    private BibDatabaseContext bibDatabaseContext;
+    private List<IntegrityMessage> messages;
+    MetaData metaData = mock(MetaData.class);
+    //private FilePreferences f;
+    //Map<Field, String> hash;
+
+    @BeforeEach
+    void setUp() {
+        //hash.put(StandardField.ABSTRACT, "");
+        //f = new FilePreferences("", hash, true, "", "");
+        bibDatabaseContext = new BibDatabaseContext();
+        checker = new FileChecker(bibDatabaseContext, mock(FilePreferences.class));
```

Comment:
Is this checker ever used in this class?

I would propose to

a) remove this unused code
b) readd the test `void testFileChecks() {` in the IntegrityCheck class as the DIFFERENCE to this test is that the `FILE` field is checked - and this coudl have some side effects.

More background:

I wondered why metaData has to be mocked. The only reason can be the special treatment of the file field. I think, we should keep this code. The code can be left in the IntegrityCheckerTest class - if we don't come up with some better place.